src/java.net.http/share/classes/java/net/http/HttpClient.java

```diff
@@ -130,9 +131,23 @@
  * privileges, should do so within an appropriate {@linkplain
  * AccessController#doPrivileged(PrivilegedAction) privileged context}.
  *
+ * @apiNote
+ * <p id="closing"> Resources allocated by the {@code HttpClient} may be
+ * reclaimed early by {@linkplain #close() closing} the client.
+ * An {@code HttpClient} instance typically manages its own pools of
+ * connections, which it can typically reuse when another request is made to
+ * the same origin server using the same protocol version and security,
+ * through the same client. In the JDK implementation, connection pools
+ * are not shared between {@code HttpClient} instances. Creating a new client
+ * for each operation, though possible, will prevent reusing such connections.
```

Comment:
I think, the reason why this sentence about creating a new connection for each operation, is here is to discourage applications from doing that. However, I found it a bit odd to have it in the middle of a section named "closing". 
The existing javadoc of this `HttpClient` has this sentence:
> Once built, an {@code HttpClient} is immutable, and can be used to send multiple requests.

Perhaps we should add this sentence there and emphasise that creating a new client for each operation is discouraged since that prevents reuse of existing connections?

Reply:
Good suggestion. I'm moving that in the paragraph following the one you quoted, since that paragraph already talks about sharing resources. I have softened the wording though, since the text is moved from a non-normative section to a normative section.